appveyor.yml

```diff
@@ -1,33 +1,37 @@
 # version is set in tools\appveyor.psm1 - Invoke-AppVeyorInstall
+environment:
+  POWERSHELL_TELEMETRY_OPTOUT: 1
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1  # avoid expensive initialization of dotnet cli, see: http://donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds
+  matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndPackaging
+  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    Purpose: BuildingAndTesting
 
-image: Visual Studio 2017
+# Stop all jobs in the matrix if any one them fail
+matrix:
+  fast_finish: true
```

Comment:
If we split jobs should we fast stop?

Reply:
The idea was: if the build itself is broken, then both jobs will fails at the same stage because they both need to build. The only undesired case would be when adding/updating new 3rd party packages and packaging is broken then the tests would abort early.

Reply:
Thanks! 

Closed.